packages/zowe-explorer/src/globals.ts

```diff
@@ -23,7 +23,7 @@ export let USS_DIR: string;
 export let DS_DIR: string;
 export let CONFIG_PATH; // set during activate
 export let LOG: imperative.Logger;
-export const COMMAND_COUNT = 121;
+export const COMMAND_COUNT = 125;
```

Comment:
Quite a few commands were added to the `package.json#contributes` property, but we only went up by 4.
It would seem that some commands are not being registered even though they might still be defined in the package.json file.

If the commands that are going away were exposed to end-users, don't we want to add something in the changelog about them going away?

Reply:
I added 3 developer commands - `zowe.onDatasetChanged, zowe.onUssChanged, zowe.onJobChanged`
- these are not added to the `contributes#commands` section of `package.json` because they don't have a title, and we don't want them in the command palette
- I noticed that I did set them to explicitly never show in the `commandPalette` section, but I don't believe that's necessary since they are not added to the first section I mentioned. Removed in upcoming commit

The other 3 commands are `zowe.diff.useLocalContent, zowe.diff.useRemoteContent, zowe.placeholderCommand`. I've added all of these to the `expectedCommands` array and those 3 are registered in that `contributes` section. 

I removed the `zowe.jobs.zosJobsOpenSpool` and `zowe.ds.ZoweNode.openPS` commands, so we went down by 2 - that leaves us at 125 commands 😋 

In an upcoming commit, I added changelog entries for the removal of both commands w/ info on how to build URIs to open data sets/jobs in v3 - sorry for not having added them previously 😅